README.md

```diff
@@ -0,0 +1,311 @@
+Install packages:
+```console
+cd backend
+yarn install
+```
+
+it is advised to use our frontend which can get cloned from and follow the guide that is written.
+```console
+git clone https://github.com/5-semesterprojekt/Frontend.git
+```
+firebaseConfig-example.ts
+
+
+After that is is also advised to change the example file and rename them without example at the end, at the moment there 3 files that has to change.
+first one is firebaseConfig-example.ts
+
+This one is simple, go to https://console.firebase.google.com and create a new project and copy paste youre own firestore config and copypaste it in the firebaseConfig-example.ts file and rename the file to firebaseConfig.ts
+
+gmailSecrets-example.ts.
+
+if you allready have a gmail then you should just follow the intructions in the file and afterwards rename it to gmailSecrets.ts
+if you dont own an gmail then create one and follow the instructions in the     
+gmailSecrets-example.ts. 
+
+jwtSecretKey-example.ts
+
+Just rename the one that is called "SECRET_KEY" to something secure since this line is used store the users token and could be hacked if compremized.
+
+
+<br>
+
+# :rocket: Start
+
+The backend can be started by running the start script:
+
+```console
+yarn start
+```
+You can make youre own website or use our frontend that we have developed along with this backend. you can use our frontend from here https://github.com/5-semesterprojekt/Frontend.git
+# 📜 Scripts
+
+### `yarn start`
+Starts the api, you would now be able to make calls, look at the api doc
+
+### `yarn test`
+Runs all tests (see [jest](https://jestjs.io) for more).
+
+### `yarn martin`
+**ToDo cahnge name**
+Formats the code according to the [.prettierrc](https://github.com/5-semesterprojekt/Backend/blob/main/.prettierrc).
+
+<br>
+# :building_construction: Contribution
+
+1. Make a branch from `main` using the suggested branch name from [Shortcut](https://app.shortcut.com/5-semester/stories/space/19/everything?team_scope_id=v2%3At%3A6536343c-3b19-48f3-96bd-e44481a7aefc%3A6536343c-ab85-4346-9338-ad967260f782). (i.e. feature/sc-{story number}/{feature-name})
+2. Commit until the feature is "complete"
+3. Run `yarn prettier` so code is formatted correctly
+4. Make pull request to `main` and request a review
+5. <ins>**Squash and merge**</ins> when all requirements are met
+
+It is **HIGHLY** advisable, that you don't branch off secondary branches. Only branch off `main`.
+
+# API Documentation
+
+### Create Event
+
+Create a new event by making a POST request to the following endpoint:
+**Needs to be logged in/auth token**
+
+```http
+POST /:orgId/
+```
+#### Request Body
+**title** (string, required): The title of the event.
+
+**start** (date, required): The start date and time of the event in ISO format (e.g., "2019-12-17T03:24:00").
+
+**end** (date, required): The end date and time of the event in ISO format.
+
+**description** (string, optional): A description of the event.
+
+**Example:**
+
+```typescript
```

Comment:
Why a typescript example instead of json like you did later?

Reply:
to much copy pasta :)